packages/@magic-sdk/react-native/src/react-native-webview-controller.tsx

```diff
@@ -138,7 +139,22 @@ export class ReactNativeWebViewController extends ViewController {
       (event.nativeEvent.url === `${this.endpoint}/send/?params=${encodeURIComponent(this.parameters)}` ||
         event.nativeEvent.url === `${this.endpoint}/send/?params=${this.parameters}`)
     ) {
-      const data: any = JSON.parse(event.nativeEvent.data);
+      // Special parsing logic when dealing with TypedArray in the payload
+      // Such change is required as JSON.stringify will manipulate the object and cause exceptions during parsing
+      // The typed Array is stringified in Mgbox with a flag as notation.
+      const data: any = JSON.parse(event.nativeEvent.data, (key, value) => {
```

Comment:
Can you expand a bit more here on what the issue with stringy is? What do you mean with manipulate the object? What is the format here?

Reply:
Nvm, I think I grok it from other PR.